scripts/utility/get_groups.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import urllib
+import sys
+from bs4 import BeautifulSoup
+
+address="http://gernot-katzers-spice-pages.com/character_tables/%s.html?fmt=simple"
+
+def clean_up(text):
+    soup = BeautifulSoup(text, "lxml")
+    pre = soup.pre
+    group = pre.b.get_text()
+    sop = {}
+    irred = {}
+    irred_count = 0
+    sop_count = 0
+    for span in pre.find_all('span'):
+        cls = span.get('class')
+        if cls == ['sop']:
+          a = span.decode_contents() 
+          if a not in sop:
+            sop[a] = sop_count
+            sop_count += 1
+        elif cls == ['irred']:
+          a = span.decode_contents() 
+          if a not in irred:
+            irred[a] = irred_count
+            irred_count += 1
+    table = [ [] for j in sop ]
```

Comment:
table seems unused

Reply:
Yes. This script was created to fetch all the character tables in data/Symmetry. It is put here for backup, but it will not be used any more.